src/pages/networks/panels/EditLocalPeeringPanel.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+  ActionButton,
+  Button,
+  ScrollableContainer,
+  SidePanel,
+  Spinner,
+  useNotify,
+  useToastNotification,
+} from "@canonical/react-components";
+import type { FC } from "react";
+import usePanelParams from "util/usePanelParams";
+import { useFormik } from "formik";
+import NotificationRow from "components/NotificationRow";
+import { queryKeys } from "util/queryKeys";
+import { useQueryClient } from "@tanstack/react-query";
+import { pluralize } from "util/instanceBulkActions";
+import type { LxdNetwork, LxdNetworkPeer } from "types/network";
+import ResourceLabel from "components/ResourceLabel";
+import NetworkLocalPeeringForm from "../forms/NetworkLocalPeeringForm";
+import type { LocalPeeringFormValues } from "../forms/NetworkLocalPeeringForm";
+import { useLocalPeering } from "context/useLocalPeerings";
+import { updateNetworkPeer } from "api/network-local-peering";
+
+interface Props {
+  network: LxdNetwork;
+}
+const EditLocalPeeringPanel: FC<Props> = ({ network }) => {
+  const panelParams = usePanelParams();
+  const { project, localPeering } = panelParams;
+  const notify = useNotify();
+  const toastNotify = useToastNotification();
+  const queryClient = useQueryClient();
+  const closePanel = () => {
+    panelParams.clear();
+    notify.clear();
+  };
+
+  const {
+    data: localPeer,
+    error,
+    isLoading,
+  } = useLocalPeering(network, project, localPeering ?? "");
+
+  const handleSuccess = () => {
+    toastNotify.success(
+      <>
+        Local peering{" "}
+        <ResourceLabel type="peering" value={localPeering ?? ""} /> updated.
```

Comment:
better use the resource link here and also mention the network as a resource link.